File: backend/components/metiers/administration_publique/guides/core/accessibility/samples/sample_fixture.test.py
"""
Test unitaire pour sample_fixture.py (guides/core/accessibility/samples)
"""
from .sample_fixture import sample_accessibility_guide

def test_sample_accessibility_guide():
    fixture = sample_accessibility_guide()
    assert isinstance(fixture, dict)
    assert fixture["name"] == "Sample Accessibility Guide"
    assert fixture["type"] == "accessibility"
    assert fixture["status"] == "active"

File: backend/components/metiers/administration_publique/guides/core/accessibility/samples/sample_fixture.test.js
// sample_fixture.test.js - Test unitaire pour sample_fixture.js (guides/core/accessibility/samples)
const sampleFixture = require('./sample_fixture');
describe('guides/core/accessibility/samples/sample_fixture.js', () => {
  it('doit exposer un objet sample correct', () => {
    expect(sampleFixture.sample).toBeDefined();
    expect(sampleFixture.sample.name).toBe('Sample Accessibility Guide');
    expect(sampleFixture.sample.type).toBe('accessibility');
    expect(sampleFixture.sample.status).toBe('active');
  });
});
